File: backend/src/main/java/org/iesvdm/musicallyx/dto/ReservaTablaDTO.java
package org.iesvdm.musicallyx.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import java.util.Date;

@Data
@AllArgsConstructor
public class ReservaTablaDTO {
    private String nombreAlumno;
    private String nombreClase;
    private String profesorClase;
    private String email;
    private Integer idReserva;
    private String estado;
    private Date fechaReserva;
    private String metodoPago;
}

File: frontend/src/app/services/reservaTablaDTO.ts
export interface ReservaTablaDTO {
  nombreAlumno: string;
  nombreClase: string;
  profesorClase:string;
  idReserva: number;
  estado: string;
  fechaReserva: string; // o Date si quieres manejarlo como Date
  metodoPago: string;
  email: string;
  precio: number;
}
